.github/dockerfiles/e2e/docker-compose.yaml

```diff
@@ -0,0 +1,46 @@
+services:
+  sproxyd:
+    profiles: ['s3c']
```

Comment:
profile is set on every container → not needed?

Reply:
I added it for future work when running the same tests with a Zenko configuration, in which containers like sproxyd and metadata won't be started.